cli/src/app.rs

```diff
@@ -174,17 +186,22 @@ async fn run_cmd(
 
     if hedging.enabled {
         println!("Starting hedging process");
+
         let hedging_send = send.clone();
         let pubsub = pubsub.clone();
         let (snd, recv) = futures::channel::mpsc::unbounded();
         checkers.insert("hedging", snd);
-        handles.push(tokio::spawn(async move {
-            let _ = hedging_send.try_send(
-                hedging::run(recv, hedging.config, okex, pubsub)
-                    .await
-                    .context("Hedging error"),
-            );
-        }));
+        let exchanges = exchanges.clone();
```

Comment:
If you destructure `exchanges` you don't have to clone the whole struct:
```
let ExchangesConfig { okex: okex_cfg, kollider: kollider_cfg } = exchanges;
```
then later
```
if let Some(okex_cfg) = okex_cfg
```
or something like that.